appserver/extras/embedded/web/src/main/resources/config/domain.xml

```diff
@@ -227,18 +227,19 @@
                 -->
                 <jvm-options>-Dfelix.fileinstall.disableConfigSave=false</jvm-options>
                 <!-- End of OSGi bundle configurations -->
-                <jvm-options>-XX:NewRatio=2</jvm-options>
                 <!-- Woodstox property needed to pass StAX TCK -->
                 <jvm-options>-Dcom.ctc.wstx.returnNullForDefaultNamespace=true</jvm-options>
-                <jvm-options>-Djdk.tls.rejectClientInitiatedRenegotiation=true</jvm-options>
                 <jvm-options>-Dorg.jboss.weld.serialization.beanIdentifierIndexOptimization=false</jvm-options>
                 <jvm-options>-Dorg.glassfish.grizzly.DEFAULT_MEMORY_MANAGER=org.glassfish.grizzly.memory.HeapMemoryManager</jvm-options>
                 <jvm-options>-Dorg.glassfish.grizzly.nio.DefaultSelectorHandler.force-selector-spin-detection=true</jvm-options>
-                <!-- Grizzly NPN Jar version -->
+                <jvm-options>[|8]-Djava.endorsed.dirs=${com.sun.aas.installRoot}/modules/endorsed${path.separator}${com.sun.aas.installRoot}/lib/endorsed</jvm-options>
+                <jvm-options>[|8]-Djava.ext.dirs=${com.sun.aas.javaRoot}/lib/ext${path.separator}${com.sun.aas.javaRoot}/jre/lib/ext${path.separator}${com.sun.aas.instanceRoot}/lib/ext</jvm-options>
+                <!-- Grizzly NPN Bootstrap compatible with used JDK version -->
                 <jvm-options>[1.8.0|1.8.0u120]-Xbootclasspath/p:${com.sun.aas.installRoot}/lib/grizzly-npn-bootstrap-1.6.jar</jvm-options>
                 <jvm-options>[1.8.0u121|1.8.0u160]-Xbootclasspath/p:${com.sun.aas.installRoot}/lib/grizzly-npn-bootstrap-1.7.jar</jvm-options>
                 <jvm-options>[1.8.0u161|1.8.0u190]-Xbootclasspath/p:${com.sun.aas.installRoot}/lib/grizzly-npn-bootstrap-1.8.jar</jvm-options>
-                <jvm-options>[1.8.0u191|1.8.0u500]-Xbootclasspath/p:${com.sun.aas.installRoot}/lib/grizzly-npn-bootstrap-1.8.1.jar</jvm-options>
```

Comment:
Except ordering here I changed also filter from 1.8.0u500 to 1.8.0u250 just as it is in other domain.xml files and added non-api.